adapters/rubicon.go

```diff
@@ -97,13 +143,24 @@ func (a *RubiconAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *
 		}
 		impExt := rubiconImpExt{RP: rubiconImpExtRP{ZoneID: params.ZoneId}}
 		rpReq.Imp[i].Ext, err = json.Marshal(&impExt)
-		bannerExt := rubiconBannerExt{RP: rubiconBannerExtRP{MIME: "text/html"}}
-		if len(params.Sizes) > 0 {
-			bannerExt.RP.SizeID = params.Sizes[0]
-			rpReq.Imp[i].Banner.Format = nil
-			rpReq.Imp[i].Banner.W = 0
-			rpReq.Imp[i].Banner.H = 0
+
+		primarySizeID := int(0)
+		altSizeIDs := []int{}
```

Comment:
We can initialize the map size to the correct size since it is known.  This way we can avoid unnecessary allocations.